README.md

```diff
@@ -1,3 +1,45 @@
-# resin.io docs
+Resin Documentation
+---------
 
-This repo contains our public-facing documentation.
+[![npm version](https://badge.fury.io/js/docs.svg)](http://badge.fury.io/js/docs)
+[![dependencies](https://david-dm.org/resin-io/docs.png)](https://david-dm.org/resin-io/docs.png)
+[![Build Status](https://travis-ci.org/resin-io/docs.svg?branch=master)](https://travis-ci.org/resin-io/docs)
+[![Build status](https://ci.appveyor.com/api/projects/status/qbsivehgnq0vyrrb?svg=true)](https://ci.appveyor.com/project/fedealconada/resin-docs)
+
+Join our online chat at [![Gitter chat](https://badges.gitter.im/resin-io/chat.png)](https://gitter.im/resin-io/chat)
+
+The official JavaScript [Resin.io](https://resin.io/) Documentation.
+
+Role
+----
+
+The intention of this repo is to provide our public-facing documentation.
+
+
+Installation
+------------
+
+Install the Resin Documentation by running:
+
+```sh
+git clone https://github.com/resin-io/docs
+cd docs
+npm install && bower install
+npm start
+```
+
+Support
```

Comment:
I don't think a "Support" section is worth here since this is not an app people will be cloning and using themselves. I'd probably extend the "Contribute" section to write up something like: "If you think something is not documented, or can be improved, don't hesitate in sending a PR!".